### DISCUSSION TITLE
Unable to open Data-Importer configs

### DISCUSSION THREAD

**Discussion by dpfahlbusch** [pimcore/pimcore#12956] [Q&A / Support]
Hey :)
Currently I can't open any data-importer configs (GraphQL configs working fine).
I get the following error:
![image](https://user-images.githubusercontent.com/22637136/186346005-f82faadd-0675-49af-85b9-e25c5555f0da.png)

Maybe someone can help :)

**Comment by mcop1** [comment]
Hello @dpfahlbusch ,
could you please post the configuration you try to open? You can find it in the settings-store or in the yaml files, depending on your configuration.
